apps/ssr-tests/webpack.config.js

```diff
@@ -10,9 +10,9 @@ module.exports = resources.createConfig('ssr-tests', false, {
 
   target: 'node',
 
-  node: {
-    fs: 'empty',
-  },
+  // node: {
```

Comment:
Can you remove this if it turns out to be unneeded?

Reply:
removed.